### ISSUE TITLE
Missing quote in extracted text if text in gettext ends with quote

### ISSUE THREAD

**Issue by kosmos342** [EnTeQuAk/django-babel-underscore#2]
```
<%= gettext('name "John"') %> 
```

it extract as 

```
name "John
```

without last quote

**Comment by kosmos342** [comment]
also it doesn't correct extract

```
<%= gettext("name \"John\"") %> 
```

**Comment by EnTeQuAk** [comment]
Fixed and released in `0.4.2`

**Comment by EnTeQuAk** [comment]
Thanks a lot!
